creator-node/src/components/healthCheck/healthCheckComponentService.js

```diff
@@ -19,7 +19,7 @@ const MIN_FILESYSTEM_SIZE = 1950000000000 // 1950 GB of file system storage
  * @param {string?} randomBytesToSign optional bytes string to be included in response object
  *    and used in signature generation
  */
-const healthCheck = async ({ libs } = {}, logger, sequelize, getMonitors, numberOfCPUs, randomBytesToSign = null) => {
+const healthCheck = async ({ libs } = {}, logger, sequelize, getMonitors, numberOfCPUs, getTranscodeQueueJobs, randomBytesToSign = null) => {
```

Comment:
i assume you meant to change `healthCheckController` call to this function and set this param to false? either way i think we should just revert this change

Reply:
ah since you never consume it i guess you actualy meant to revert this

Reply:
oh yeah leftover relic, will remove

Reply:
cool